packages/chrome/src/styled/nav/StyledNavList.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.nav_list';
+
+export const StyledNavList = styled.ul.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  order: -1;
```

Comment:
CSS `order` rules can lead to accessibility problems and I've had mixed experiences with it. Do we still need these rules or should we leave it to the user to specify the correct order (potential breaking change 💥)?

https://developer.mozilla.org/en-US/docs/Web/CSS/order#accessibility_concerns
https://www.w3.org/TR/css-flexbox-1/#order-accessibility

Reply:
Actually, you just jogged my senses a bit. Since this element is just a `ul` and it's a container, it naturally shouldn't land in any tab/render order. So, we can probably just remove it I think. 🤔 

but your concern is valid and I don't think we need to include `order` in these elements since the original tab oder already prescribes the correct navigation behavior. I'll give it a quick test to confirm, and we can discuss inconsistencies if there are any!

Reply:
I think I see why `order` is used here.

It's to ensure that a child composition of:

```jsx
<Nav>
  <Nav.List>
    <Nav.Item hasBrandmark />
    <Nav.Item>
      <Nav.ItemText />
      <Nav.ItemIcon />
    </Nav.Item>
    <Nav.Item hasLogo />
  </Nav.List>
</Nav>
```

Renders in the expected way in the DOM (using React components for simplicity):

```jsx
<nav>
  <ul>
    <li class="StyledLogoItem />
    <li class="StyledItem">
      <span class="StyledItemIcon" />
      <span class="StyledItemText" />
    </li>
    <li class="StyledBrandmarkItem" />
  </ul>
</nav>
```

E.g., a break-prevention measure. I can't speak to if this is still desirable though. I assume so but maybe JZ or design have further insights. I would agree it is a bit heavy-handed but I can understand the logic after toying with it.

Reply:
Thanks for your excellent explanation. 💯  It looks like removing `order` isn't so trivial after all and definitely falls out of scope for this PR. Perhaps something we can revisit in the future.

Reply:
Yeah, I agree on revisiting if it makes sense!

Reply:
The `order` CSS throughout Chrome is intentional, and despite the a11y concerns we keep it because enforcing consistent UX tab ordering despite consumer usage is the more important concern. Perhaps if/when a grid layout vs. flex is explored in the future, we can reevaluate.